src/genophenocorr/model/_variant.py

```diff
@@ -331,6 +331,7 @@ def variant_key(self) -> str:
         Get a readable representation of the variant's coordinates.
 
         For instance, ``X_12345_12345_C_G`` for a sequence variant or ``22_10001_20000_INV`` for a symbolic variant.
+        If the 'ref' or 'alt' sequences are over 50 bases, they are replaced with the number of bases. Example: ``16_78386858_78425054_--38197bp--_A``
```

Comment:
The code of this function not doing what is advertised.
In other words, the pydoc suggests some behavior, but the code does something else.